website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, host_page, host_name, mode, options) {
```

Comment:
Why do `host_page`, `host_name`, and `mode` need to be in here? They don't seem to be used by CommentPane at all.